diff --git a/CarRental/CarRental.Console/CarRental.ConsoleUI.csproj b/CarRental/CarRental.Console/CarRental.ConsoleUI.csproj
index cdf0cfb..04a5153 100644
--- a/CarRental/CarRental.Console/CarRental.ConsoleUI.csproj
+++ b/CarRental/CarRental.Console/CarRental.ConsoleUI.csproj
@@ -7,6 +7,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\CarRental.DAL\CarRental.DAL.csproj" />
     <ProjectReference Include="..\CarRental.Logic\CarRental.Logic.csproj" />
diff --git a/CarRental/CarRental.Console/Program.cs b/CarRental/CarRental.Console/Program.cs
index e2298e7..746e63a 100644
--- a/CarRental/CarRental.Console/Program.cs
+++ b/CarRental/CarRental.Console/Program.cs
@@ -1,23 +1,12 @@
-<<<<<<< HEAD
 ﻿using CarRental.DAL;
 
 namespace CarRental.ConsoleUI;
 internal class Program
-=======
-﻿namespace CarRental.ConsoleUI
->>>>>>> origin
 {
     static void Main(string[] args)
     {
-<<<<<<< HEAD
         Console.Title = AppConfig.APP_NAME;
 
         MainMenu.StartMenu();
-=======
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
->>>>>>> origin
     }
 }
diff --git a/CarRental/CarRental.DAL/CarRental.DAL.csproj b/CarRental/CarRental.DAL/CarRental.DAL.csproj
index bd833f7..3ae3cad 100644
--- a/CarRental/CarRental.DAL/CarRental.DAL.csproj
+++ b/CarRental/CarRental.DAL/CarRental.DAL.csproj
@@ -6,7 +6,7 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-<ItemGroup>
+  <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
   </ItemGroup>
 
diff --git a/CarRental/CarRental.DAL/Class1.cs b/CarRental/CarRental.DAL/Class1.cs
deleted file mode 100644
index 289182c..0000000
--- a/CarRental/CarRental.DAL/Class1.cs
+++ /dev/null
@@ -1,7 +0,0 @@
-﻿namespace CarRental.DAL
-{
-    public class Class1
-    {
-
-    }
-}
\ No newline at end of file
diff --git a/CarRental/CarRental.Logic/CarRental.Logic.csproj b/CarRental/CarRental.Logic/CarRental.Logic.csproj
index f62759a..a29ba75 100644
--- a/CarRental/CarRental.Logic/CarRental.Logic.csproj
+++ b/CarRental/CarRental.Logic/CarRental.Logic.csproj
@@ -6,6 +6,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\CarRental.DAL\CarRental.DAL.csproj" />
   </ItemGroup>
diff --git a/CarRental/CarRental.Logic/SearchViewModelDto.cs b/CarRental/CarRental.Logic/SearchViewModelDto.cs
index 0180523..c35c83b 100644
--- a/CarRental/CarRental.Logic/SearchViewModelDto.cs
+++ b/CarRental/CarRental.Logic/SearchViewModelDto.cs
@@ -1,10 +1,17 @@
-﻿namespace CarRental.Logic;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
-public class SearchViewModelDto
+namespace CarRental.Logic
 {
-    public string ModelAndMake { get; set; }
-    public int ProductionYearFrom { get; set; }
-    public int ProductionYearTo { get; set; }
-    public DateTime StartDate { get; set; }
-    public DateTime EndDate { get; set; }
+    public class SearchViewModelDto
+    {
+        public string ModelAndMake { get; set; }
+        public int ProductionYearFrom { get; set; }
+        public int ProductionYearTo { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+    }
 }
diff --git a/CarRental/CarRental.Web/Models/SearchViewModel.cs b/CarRental/CarRental.Web/Models/SearchViewModel.cs
index ff1369e..2309541 100644
--- a/CarRental/CarRental.Web/Models/SearchViewModel.cs
+++ b/CarRental/CarRental.Web/Models/SearchViewModel.cs
@@ -1,4 +1,5 @@
-﻿using CarRental.Logic;
+﻿using CarRental.DAL.Models;
+using CarRental.Logic;
 
 namespace CarRental.Web.Models
 {
diff --git a/CarRental/CarRental.Web/Views/Search/Index.cshtml b/CarRental/CarRental.Web/Views/Search/Index.cshtml
index b486066..45b7663 100644
--- a/CarRental/CarRental.Web/Views/Search/Index.cshtml
+++ b/CarRental/CarRental.Web/Views/Search/Index.cshtml
@@ -43,7 +43,14 @@
         </form>
         @foreach (var item in Model.Cars)
         {
-         <partial name="_CarTiles", model="item" />
+        <div class="card" style="width: 18rem;">
+                <img class="card-img-top" src="images/car-1.jpg" alt="Card image cap">
+            <div class="card-body">
+                    <h5 class="card-title">@item.CarModel title</h5>
+                <p class="card-text">@item.Make @item.Id</p>
+                <a href="#" class="btn btn-primary">Go somewhere</a>
+            </div>
+        </div>
         }
     </div>
 </div>
diff --git a/CarRental/CarRental.Web/Views/Shared/_CarTiles.cshtml b/CarRental/CarRental.Web/Views/Shared/_CarTiles.cshtml
deleted file mode 100644
index 792a847..0000000
--- a/CarRental/CarRental.Web/Views/Shared/_CarTiles.cshtml
+++ /dev/null
@@ -1,17 +0,0 @@
-﻿@model IEnumerable<CarRental.Web.Models.CarListModel>
-
-@{
-
-}
-
- @foreach (var item in Model)
-    {
-        <div class="card" style="width: 18rem;">
-                <img class="card-img-top" src="images/car-1.jpg" alt="Card image cap">
-            <div class="card-body">
-                    <h5 class="card-title">@item.CarModel title</h5>
-                <p class="card-text">@item.Make @item.Id</p>
-                <a href="#" class="btn btn-primary">Go somewhere</a>
-            </div>
-        </div>
-    }
\ No newline at end of file
